Sources/PointFree/VideoJs.swift

```diff
@@ -10,18 +10,61 @@ var videoJsHead: [ChildOf<Element.Head>] {
       rel(.stylesheet)
       ]),
     .init(script([
+      .init("onload", "videoJsLoaded()"),
```

Comment:
since the loading of video.js is deferred we need to be notified when it is done loading

Reply:
Nice. We can swap this out for a proper function when the **swift-html** work merges.